src/elements/content-explorer/ContentExplorer.js

```diff
@@ -314,6 +324,46 @@ class ContentExplorer extends Component<Props, State> {
         }
     }
 
+    /**
+     * Metadata queries success callback
+     *
+     * @private
+     * @param {Object} metadataQueryCollection - Metadata query response collection
```

Comment:
Should this parameter have the type `MetadataQueryResponse` instead of `Object`?

Reply:
This is JSDoc param type like other functions.